macros/get_metafields.sql

```diff
@@ -37,8 +53,8 @@ final as (
         {% endfor %}
     from source_table
     left join lookup_object 
-        on lookup_object.{{ reference_field }}_id = source_table.{{ reference_value }}_id
-        and lookup_object.{{ reference_field }} = '{{ reference_value }}'
+        on lookup_object.owner_resource_id = source_table.{{ id_column }}
```

Comment:
I actually think we should keep the old approach for the owner_resource_id where it was `{{ reference_field }}_id`. I imagine it was parameterized for a reason and we should probably maintain that instead of hard coding `owner_resource_id`.

Let me know if you think differently.

Reply:
Sounds good, applied!